qt/Formatter.cc

```diff
@@ -24,46 +24,36 @@ namespace
  unsigned int size_K;
 }
 
-QString Formatter::unitStrings[3][5];
+QString Formatter::unitStrings[3][5] = {
+    { {tr("B/s")}, {tr("kB/s")}, {tr("MB/s")}, {tr("GB/s")}, {tr("TB/s")}},
+    { {tr("B")},   {tr("kB")},   {tr("MB")},   {tr("GB")},   {tr("TB")}},
+    { {tr("B")},   {tr("KiB")},  {tr("MiB")},  {tr("GiB")},  {tr("TiB")}},
+};
 
 void
 Formatter::initUnits ()
 {
+
  speed_K = 1000;
- unitStrings[SPEED][B]  = tr ( "B/s");
- unitStrings[SPEED][KB] = tr ("kB/s");
- unitStrings[SPEED][MB] = tr ("MB/s");
- unitStrings[SPEED][GB] = tr ("GB/s");
- unitStrings[SPEED][TB] = tr ("TB/s");
  tr_formatter_speed_init (speed_K,
-                          unitStrings[SPEED][KB].toUtf8().constData(),
-                          unitStrings[SPEED][MB].toUtf8().constData(),
-                          unitStrings[SPEED][GB].toUtf8().constData(),
-                          unitStrings[SPEED][TB].toUtf8().constData());
+                          qPrintable(unitStrings[SPEED][KB]),
```

Comment:
Did you mean `qUtf8Printable()` (sadly it's only available since Qt 5.4)? Otherwise the transformation isn't equivalent (`qPrintable()` calls `toLocal8Bit()`, not `toUtf8()`) and you should leave it as it was before (or define `qUtf8Printable()` for Qt <5.4).